services/postgres/Dockerfile

```diff
@@ -0,0 +1,5 @@
+ENV POSTGRES_USER=tugboat \
```

Comment:
This sets `tugboat` as the postgres superuser account. I think we should mirror what we already do with the mysql & family images instead.

* Restrict the default superuser account to access from the database container itself
* Create a non-privileged `tugboat` user with full access to the `tugboat` database

Looks like the way to do that is to add an init script to the image. See "Initialization scripts" at https://hub.docker.com/_/postgres